examples/edge_intelligence/Readme.md

```diff
@@ -0,0 +1,20 @@
+This folder contains example files for different models: pfa, sklearn, tensorflow, rule.
+
+# RuleEdgeComponent
+
+Rule edge component is like different edge components provided by Liota, you can define a simple rule depending on your requirement as a lambda function and the rule edge component will process the incoming metrics and give output to the actuator on the basis of the rule you have defined as lambda function.
+
+# Example
+If you have to take rpm as a metric and if rpm exceeds certain limits you have to perform some action, you can define a lambda function as:
+```
+Rule = lambda x : 0 if (rpm>=rpm_limit) else 0
+```
+Now just pass the rule to RuleEdgeComponent along with the action_actuator and exceed_limit.
+exceed_limit is a parameter which can be specified about after how many times the limit is exceeded then action should be taken.
+example: if rpm exceeds rpm_limit consecutively 3 times then only action should be applied, in that case we would assign exceed_limit = 3
+```
+edge_component = RuleEdgeComponent(ModelRule,exceed_limit,actuator_udm=action_actuator)
+```
+After this just register the edge_component and start_collecting the values from sensor.
```

Comment:
the values which will get published to RuleEdge component. Please refer the example.